Failing to find gtest

I followed the instructions for cycstub, but I'm getting a failure to find the gtest header (which should be in the cyclus include directories... right?) Anyone know what I am missing here?

Maybe this is because all efforts to find the gtest include dirs were stripped in this commit: fadd00b70e688d1767b8a7649ebf07700124aa25 ? 

Maybe not. I'm pretty sure I was just following the instructions. I cloned cycstub, did the rename, and called python install.py --prefix=../install . That install directory is where my version 1 cyclus and cycamore are both (successfully) installed.

FWIW, I pushed it up to https://github.com/katyhuff/fco_transition .

```
Linking CXX shared library ../lib/cyclus/Fco_transitionFacility/libFco_transitionFacility.dylib
[  9%] Built target Fco_transitionFacility
Scanning dependencies of target Fco_transitionFacility_unit_tests
[ 14%] Building CXX object src/CMakeFiles/Fco_transitionFacility_unit_tests.dir/Users/khuff/repos/cyclusv1/cyclus/share/cyclus_default_unit_test_driver.cc.o
/Users/khuff/repos/cyclusv1/cyclus/share/cyclus_default_unit_test_driver.cc:5:10: fatal error: 'gtest/gtest.h' file not found
#include <gtest/gtest.h>
         ^
1 error generated.
make[2]: *** [src/CMakeFiles/Fco_transitionFacility_unit_tests.dir/Users/khuff/repos/cyclusv1/cyclus/share/cyclus_default_unit_test_driver.cc.o] Error 1
make[1]: *** [src/CMakeFiles/Fco_transitionFacility_unit_tests.dir/all] Error 2
make: *** [all] Error 2
Traceback (most recent call last):
  File "install.py", line 132, in <module>
    main()
  File "install.py", line 129, in main
    install(args)
  File "install.py", line 66, in install
    shell=(os.name == 'nt'))
  File "/Users/khuff/anaconda/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make']' returned non-zero exit status 2
```

--- a/cmake/FindCyclus.cmake
+++ b/cmake/FindCyclus.cmake
@@ -40,10 +40,16 @@ FIND_PATH(CYCLUS_CORE_TEST_INCLUDE_DIR agent_tests.h
   /usr/local/cyclus /opt/local/cyclus  
   PATH_SUFFIXES cyclus/include include include/cyclus include/cyclus/tests cyclus/include/tests)
 
-SET(
-  CYCLUS_CORE_TEST_INCLUDE_DIR ${CYCLUS_CORE_TEST_INCLUDE_DIR}
-  ${CYCLUS_CORE_TEST_INCLUDE_DIR}/../gtest
-  )
+# find the directory containing gtest
+FIND_PATH(CYCLUS_GTEST_INCLUDE_DIR gtest/gtest.h
+  HINTS "${CYCLUS_ROOT_DIR}" "${CYCLUS_ROOT_DIR}"
+  "${CYCLUS_ROOT_DIR}/include"
+  "${CYCLUS_ROOT_DIR}/include/cyclus"
+  /usr/local/cyclus /opt/local/cyclus
+  PATH_SUFFIXES cyclus/include include include/cyclus)
+
+SET(CYCLUS_CORE_TEST_INCLUDE_DIR ${CYCLUS_CORE_TEST_INCLUDE_DIR}
+  ${CYCLUS_GTEST_INCLUDE_DIR})
 
 # Add the root dir to the hints
 SET(CYCLUS_ROOT_DIR "${CYCLUS_CORE_INCLUDE_DIR}/../..")

